lua/autorun/server/sv_doors.lua

```diff
@@ -242,7 +265,7 @@ net.Receive( "SyncCoOwner", function( len, sender )
 	local remove = net.ReadBool()
 	local door = ents.GetByIndex( index )
 	if DoorTable[index] then
-		DS_Notify( sender, "This door is managed by a group and cannot be owned." )
+		DS_Notify( sender, "This door is locked ..." )
```

Comment:
This should be following the same format as the rest of the notifications using the config you made, and it shouldn't be displaying a locked notification since this is used to detect whether or not a group is assigned to the door.
```suggestion
		DS_Notify( sender, DOOR_CONFIG_MESSAGES["Managed By Another Group"] )
```